Formula/o/openrdap.rb

```diff
@@ -0,0 +1,19 @@
+class Openrdap < Formula
+  desc "Command-line client for the Registration Data Access Protocol"
+  homepage "https://www.openrdap.org"
+  url "https://github.com/openrdap/rdap/archive/refs/tags/v0.9.1.tar.gz"
+  sha256 "06a330a9e7d87d89274a0bcedc5852b9f6a4df81baec438fdb6156f49068996d"
+  license "MIT"
+  head "https://github.com/openrdap/rdap.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"rdap"), "cmd/rdap/main.go"
+  end
+
+  test do
+    output = shell_output("#{bin}/rdap --help 2>&1", 1)
```

Comment:
We need a test that exercises the some of the functionality of the app. Version checks or usage checks (foo --version or foo --help) are not sufficient, as explained in the formula cookbook.

In most cases, a good test would involve running a simple test case: run #{bin}/foo input.txt.

- Then you can check that the output is as expected (with assert_equal or assert_match on the output of shell_output)
- You can also check that an output file was created, if that is expected: assert_predicate testpath/"output.txt", :exist?

Some advice for specific cases:

- If the formula is a library, compile and run some simple code that links against it. It could be taken from upstream's documentation / source examples.
- If the formula is for a GUI program, try to find some function that runs as command-line only, like a format conversion, reading or displaying a config file, etc.
- If the software cannot function without credentials, a test could be to try to connect with invalid credentials (or without credentials) and confirm that it fails as expected.
- Same if the software requires a virtual machine, docker instance, etc. to be running.

Reply:
Hi @SMillerDev ! 

I'm really sorry about not looked up on formula cookbook for more tests requirements. I've based the formula on pushed go projects on homebrew-core.

As you requested, i've put a few tests on formula and submitted for you review.